src/Auth/AuthContainer.tsx

```diff
@@ -0,0 +1,74 @@
+import React, {useState} from 'react';
+import {Button, TextInput} from 'react-materialize';
+import {auth} from '../Firebase/firebase';
+import firebase from 'firebase/compat/app';
+import {useNavigate} from 'react-router-dom';
+import './style/style.css';
+
+export const AuthContainer = () => {
+    const [password, setPassword] = useState('')
+    const [email, setEmail] = useState('')
+    const [user, setUser] = useState<firebase.User | null>(null)
+    const navigate = useNavigate();
+
+    function createEmail(email: string) {
```

Comment:
Naming. You are not creating an email, you just changing the state, so it could be called onEmailChange or something like that